### ISSUE TITLE
header titles not accessible

### ISSUE THREAD

**Issue by jmensch1** [react-navigation/react-navigation-stack#61]
I'm setting my screen titles using the `title` property in `navigationOptions`, and the titles aren't accessible to VoiceOver. 

I noticed that the `accessible` prop is explicitly set to `false` on the [HeaderTitle component](https://github.com/react-navigation/react-navigation-stack/blob/master/src/views/Header/HeaderTitle.js). If I change it to `true`, the titles become accessible. 

I'd submit a PR, but the prop was specifically set to `false` in this [commit](https://github.com/react-navigation/react-navigation-stack/commit/1931b5b9688007fa1316aef0023b6546a931e387). So I'm not sure if maybe there's some configuration where it's necessary for the titles not to be accessible.

**Comment by MagTuxGit** [comment]
I added `HeaderTitle` component from the repo to my project and removed `accessible` property.
In `navigationOptions` I use it this way:
```
// title: 'Screen Title'
headerTitle: <HeaderTitle style={{color: "white"}}>Screen Title</HeaderTitle>
```

**Comment by adrienbataille** [comment]
I got the same problem after upgrading from 2.0.1 to 3.0.9. It was working before, so for me it's a regression.
